linkerd/examples/client-options.yaml

```diff
@@ -26,6 +26,10 @@ routers:
       # this client only
       loadBalancer:
         kind: aperture
+      socketOptions:
+        reusePort: true
+        readTimeoutMs: 61000
+        writeTimeoutMs: 61000
```

Comment:
Let's also add the `keepAlive` option here. It doesn't hurt to have more documentation 😄